src/cryptography/hazmat/primitives/serialization/pkcs12.py

```diff
@@ -7,3 +7,9 @@
 
 def load_key_and_certificates(data, password, backend):
     return backend.load_key_and_certificates_from_pkcs12(data, password)
+
+
+def generate_pkcs12(password, name, key, cert, ca, key_encryption,
+                    cert_encryption, backend):
```

Comment:
Sorry, don't know how I missed this earlier -- new APIs should not take a backend parameter, they should automatically use the OpenSSL backend